src/pages/404.js

```diff
@@ -42,12 +48,12 @@ const NotFoundPage = ({ pageContext }) => {
                 line-height: 1;
               `}
             >
-              404
+              {t('404.h1')}
             </h1>
-            <p>
+            <Trans i18nKey="404.content" parent="p">
```

Comment:
Looks like the key you had above was named `404.text`. Could you make sure this matches the key?

Reply:
✅